db/migrate/20190913045600_add_display_welcome_to_maps.rb

```diff
@@ -1,4 +1,4 @@
-class AddDisplayWelcomeToMaps < ActiveRecord::Migration[5.2]
```

Comment:
Hey @katungi, I see you modified these three files, could you please remove them since they are not part of the issue?
After this we should be ready to merge. Thanks

Reply:
how do  remove them @cesswairimu ?
do i gitignore?

Reply:
Nope, on your branch do
`git log`
pick the second last SHA and do  
`git reset SHA`
if you now do `git status` you will see the your files unstaged
`git add filename` to add the files you only want 
then `git commit` then `git push -f origin branchname`, 
Makes sense? Let me know if u get stuck